artifacts/how-to/Automatic Call Scheduling/AUTOMATIC_SCHEDULING_FLOW.md

```
@@ -0,0 +1,1060 @@
+
+**What it does:**
+
+1. **Creates IAM Role**
+   ```bash
+   aws iam create-role \
+     --role-name lambda-call-initiator-execution-role \
+     --assume-role-policy-document '{
+       "Version": "2012-10-17",
+       "Statement": [{
+         "Effect": "Allow",
+         "Principal": {"Service": "lambda.amazonaws.com"},
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)